standard/ERCs/erc-6900.md

```diff
@@ -588,7 +586,9 @@ This proposal includes several interfaces that build on ERC-4337. First, we stan
 
 ## Backwards Compatibility
 
-TODO
+Existing accounts that are deployed as proxies may have the ability to upgrade account implementations to one that supports this standard for modularity. Depending on implementation logic, existing modules may be wrapped in an adapter contract to adhere to the standard.
```

Comment:
Do you think we should mention anything related to storage re: upgrades?

Technically speaking, the RI namespaces storage, but is that expected of other implementations? If not, that could cause issues with upgrades. Do you think we ought to mention a little warning about storage collisions here?

Reply:
The focus of this section is on backwards compatibility, and because the standard is proxy-agnostic (whether or not you use one at all), I'm not sure we want to introduce it here. If we explicitly call out namedspaced storage (ERC-7201) then we also have to add it to the dependencies list, and I think it's probably not worth introducing the dependency just for this section.

Maybe we just add a sentence like "Users should exercise caution when doing proxy upgrades, and verify that no issues, such as storage collisions, occur in the process." ?

Reply:
I think it's fine as is then-- better not bleed into proxies in that case!